regression/scripts/noise_fidelity_one/propeller_noise.py

```diff
@@ -65,7 +67,9 @@ def main():
     
     # Set up for Propeller Model
     prop.inputs.omega                            = np.atleast_2d(omega).T
-    conditions                                   = Aerodynamics()   
+    prop.inputs.pitch_command                    = 0.
+    conditions                                   = Aerodynamics()
+    conditions._size                             = 3
```

Comment:
_size is supposed to be a hidden item. _ is the pythonic style to indicate a user should not see this variable. This variable has existed in SUAVE since day one.

However, since this regression doesn't have a mission associated with it the data structures are never expanded. This is a hack to get the regressions to run. A user should never do this.